### DISCUSSION TITLE
How to specify general.name when quantizing?

### DISCUSSION THREAD

**Discussion by shibe2** [ggml-org/llama.cpp#6663] [Q&A] [answered]
I get:

>general.name str              = .

**Comment by phymbert** [comment]
Please detailed all steps you have followed in order to assist you.

At the moment it is not possible to override the metadata with a string `--override-kv general.name=str:myname`.

Maybe if this PR gets merged:
-  #6658

**Reply by shibe2** [reply]
My question is about a general way to set arbitrary model name in GGUF file. If this it not yet provided, then, perhaps, I need to apply a patch to allow str override?

**Reply by phymbert** [reply, marked as answer]
The model name comes from the convert python script. It is probably easier to do it at the very beginning then

**Reply by shibe2** [reply]
Okay, I modified the script.
